Add Home Assistant Cast service docs (#10339)
home-assistant/home-assistant.github.io: source/_components/cast.markdown

you don't have the discovery integration enabled, you can enable the Cast
integration by going to the Integrations page inside the config panel.

## Home Assistant Cast

> The Cast integration allows you to start Home Assistant Cast on any Chromecast device. This is done via the `cast.show_lovelace_view` service. The service takes the path of a Lovelace view and an entity ID of a Cast device to show the view on.

Reviewer: ```suggestion
The Cast integration allows you to start Home Assistant Cast on any Chromecast device, using the `cast.show_lovelace_view` service. The service takes the path of a Lovelace view and an entity ID of a Cast device to show the view on.
```

```json
{
  "entity_id": "media_player.office_display_4",
  "view_path": "lights"
}
```

> 

Reviewer: I really think we should add a small automation example here...
While for the service call the JSON works, in practice, people use those in automations and do not use JSON in those cases.

Reviewer: Let's skip this, for now, ask discussed, maybe we should add YAML support to the dev tools in the future.

{% configuration %}
media_player:

> description: A list that contains all Cast devices.

Reviewer: Indents got lost here

Reviewer: Fixed.
